ValidAnagram.py

```diff
@@ -0,0 +1,22 @@
+#
+# https://leetcode.com/problems/valid-anagram/description/
+#
+#####################################################################################################################################################
+#                                                            PROBLEM DESCRIPTION                                                                    #
+#####################################################################################################################################################
+#
+# Given two strings s and t, return true if t is an anagram of s, and false otherwise.
+# An Anagram is a word or phrase formed by rearranging the letters of a different word or phrase, typically using all the original letters exactly once.
+
+# Example 1:
+# Input: s = "anagram", t = "nagaram"
+# Output: true
+
+# Example 2:
+# Input: s = "rat", t = "car"
+# Output: false
+#
+#####################################################################################################################################################
+class Solution:
+    def isAnagram(self, s: str, t: str) -> bool:
```

Comment:
variable names!

Reply:
Leetcode has some variable names in functions defined themselves. And we receive values in them such that our solution works on it, against the test case.